R/PictureBlock.R

```diff
@@ -40,7 +40,7 @@ PictureBlock <- R6::R6Class( # nolint: object_name_linter.
           if (inherits(content, "grob")) {
             grid::grid.newpage()
             grid::grid.draw(content)
-          } else if (inherits(content, "gg")) {
+          } else if (inherits(content, c("gg", "Heatmap"))) {
```

Comment:
please add comment about origin of the heatmap 
# "Heatmap" S4 from ComplexHeatmap

Reply:
added